torch/_export/db/gen_example.py

```diff
@@ -0,0 +1,28 @@
+import os
```

Comment:
what is this file for again?

Reply:
this is a script to generate a new example file with empty code.